Source/WebCore/svg/graphics/filters/SVGFilter.cpp

```diff
@@ -235,6 +221,16 @@ FilterEffectVector SVGFilter::effectsOfType(FilterFunction::Type filterType) con
     return copyToVector(effects);
 }
 
+OptionSet<FilterMode> SVGFilter::supportedFilterModes() const
+{
+    OptionSet<FilterMode> modes;
+
+    for (auto& term : m_expression)
+        modes.add(term.effect->supportedFilterModes());
```

Comment:
Isn't the result mode the intersection of the supported modes for each effect, not their union?

Reply:
Yes you are right. Fixed.